grails-app/services/au/org/ala/merit/AbnLookupService.groovy

```diff
@@ -19,23 +20,28 @@ class AbnLookupService {
  * @return abnDetails
  */
     Map lookupOrganisationNameByABN(String organisationABN){
+        Map abnDetails
 
-        String abnLookupToken = grailsApplication.config.getProperty('abn.abnLookupToken')
-        String url = grailsApplication.config.getProperty('abn.abnUrl')
-        String abnLookupUrlString =  url + organisationABN + "&guid=" + abnLookupToken
+        if (!validateABN(getNonBlankNumericStringWithoutWhitespace(organisationABN))) {
```

Comment:
I think you've misunderstood my request from the previous review - I was requesting you move the call to "getNonBlankNumericStringWithoutWhitespace" back into this method but still use it for the actual call to the service.

e.g. String abn =  getNonBlankNumericStringWithoutWhitespace(organisationABN);
if (!validate(abn)) _{...}. else { ..callServiceWith(abn)..}

(note not literal code, just trying to be more clear about what I was expecting.